dkg-gadget/src/metrics.rs

```diff
@@ -30,10 +41,36 @@ impl Metrics {
 				Gauge::new("dkg_validator_set_id", "Current DKG active validator set id.")?,
 				registry,
 			)?,
-			propagated_messages: register(
+			dkg_propagated_messages: register(
 				Counter::new("dkg_propagated_messages", "Number of DKG messages propagated.")?,
 				registry,
 			)?,
+			dkg_inbound_messages: register(
+				Counter::new("dkg_inbound_messages", "Number of DKG messages received.")?,
+				registry,
+			)?,
+			dkg_error_counter: register(
+				Counter::new("dkg_error_counter", "Number of DKG errors generated")?,
+				registry,
+			)?,
```

Comment:
Will this provide insights into what dkg error? For instance, will it assist in detecting when a dkg keygen fails or just report an error of 1. Idk if that is exactly clear lol

Reply:
It will report how many times the `handle_dkg_error` function has been triggered, not all are critical. I think we can only track metrics here, we should rely on logs for the exact error.